setup.cfg

```diff
@@ -1,4 +1,5 @@
 [tool:pytest]
 testpaths = test
-# pytest with coverage seems to be causing issues in parallel tests
-# addopts = --cov=lyncs_io --cov-report term-missing
+
+[coverage:run]
+parallel = true
```

Comment:
Ok nice, but does it work with the previous flags? 
`addopts = --cov=lyncs_io --cov-report term-missing`
If yes, can you restore them? :)

Reply:
I'll check it out but I don't think so. The root of the issue seems to be --cov-report when run in parallel as two cores will try and write the same file. Instead you need to specify coverage explicitly as  follows:
```sh
# Individual .coverage file is generated per core
mpiexec -n 2 --oversubscribe coverage run --rcfile=setup.cfg -m mpi4py -m pytest test/parallel_tests/ --with-mpi
# Combine parallel coverage reports in single .coverage file
coverage combine
coverage xml
```

Reply:
I've tried it out and the previous flags work. I restore them

Reply:
I take it back, it happens to work but not always. Most probably there are race conditions.

Reply:
Indeed that happen to me too. From what I have seen both mpi4py and dask kind of exclude the test directories from the coverage report. Not sure though which is the right route to take.